src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,8 +14,8 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
-          <h2 class="kuiLocalTab">
+        <header ng-if="!sectionName" id="tabHeader">
+          <h2>
```

Comment:
Thanks for the heads up @cjcenizal! I'll revert this back, or possibly introduce a new class that has the same styles sans the hover treatment.